float32 heading
float32 target_heading
float32 l_distance
float32 alpha
